.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+  - "2.6"
+  - "2.7"
+  - "3.2"
+  - "3.3"
+# command to install dependencies
+script: python tests/test_all_of_the_units.py
+branches:
+  only:
+    - master
```

Comment:
@sarayd This is not related to the issue mentioned, can you please cleanup your pull request?